### PR TITLE
Add basic GUI

### PR DESCRIPTION
**Pull request by abhsag24**
adding ui as an experimental feature with `--ui` option

### CODE REVIEW

File: uiplib/GUI.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        self.updateUI()
+        # run the main event loop of UI
+        self.root.mainloop()
+
+
+    def updateUI(self):
+        # update UI with data received
+        while self.queue and not self.queue.empty():
+            pass
+        print(time.ctime()) #remove this comment
```
Comment (abhsag24) [inline uiplib/GUI.py]: i guess it's time to remove? :P

**Comment by abhsag24** [comment]
@nkprince007 can we have all requirements for this in an `gui-requirements.txt` which we will merge with the main `requirements.txt` when we start using GUI by default?

**Comment by abhsag24** [comment]
i guess that and we're good to merge this

**Comment by abhsag24** [comment]
i guess it's important to merge so we can have issues regarding GUI alone and have more people working on this for faster development

File: uiplib/GUI.py (hunk 1 above)
Comment (nkprince007) [inline uiplib/GUI.py]: Yeah, it pretty much is.

File: uiplib/GUI.py [hunk 2]
```diff
@@ -139,8 +144,11 @@ def set_wallpaper(self):
         image = self.images[self.index]
         change_background(image)
 
-    def refresh(self):
-        print("Refresh Clicked!")
+    def download(self):
+        pass
```
Comment (nkprince007) [inline uiplib/GUI.py]: I still haven't worked on it. Maybe call the scheduler to download new ones by flushing the very old ones or something like that.

**Pull request merged**